modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/session/SessionExpiredException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.sql.engine.session;
+
+import static org.apache.ignite.lang.IgniteStringFormatter.format;
+
+import org.apache.ignite.lang.IgniteInternalException;
+
+/**
+ * Thrown when session has been expired but not yet been collected by cleaner thread.
+ */
+public class SessionExpiredException extends IgniteInternalException {
+
+    private static final long serialVersionUID = 0L;
+
+    /** Constructor. */
+    public SessionExpiredException(SessionId sessionId, Throwable cause) {
+        super(format("Session has been expired [{}]", sessionId), cause);
```

Comment:
the same: this constructor is deprecated

Reply:
SessionExpiredException now extends RuntimeException.

Reply:
Replaced SessionExpiredException  with SessionNotFoundException

Reply:
As I see - this class can be removed.